src/scenario.h

```diff
@@ -89,6 +100,8 @@ typedef struct {
 	sint32 flags;				// 0x0268
 	uint32 company_value;		// 0x026C
 	char completed_by[64];		// 0x0270
+	scenario_source source_game;// new in OpenRCT2
```

Comment:
Yes, I realise it's pretty much a direct copy of the file's header. I didn't notice any problems loading or saving though, so I thought it was fine. Hmm, how else would you have me do it? The scenario list is sorted shortly after loading -- I'd rather not have to keep two arrays sorted consistently...